lib/rules/block-closing-brace-newline-before/__tests__/index.js

```diff
@@ -271,66 +292,97 @@ testRule(rule, {
   reject: [
     {
       code: "a { color: pink;\ntop: 0; }",
+      fixed: "a { color: pink;\ntop: 0;}",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 8
     },
     {
       code: "a { color: pink;\r\ntop: 0; }",
+      fixed: "a { color: pink;\r\ntop: 0;}",
       description: "CRLF",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 8
     },
     {
       code: "a { color: pink;;\ntop: 0;; }",
+      fixed: "a { color: pink;;\ntop: 0;;}",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 9
     },
     {
       code: "a { color: pink;;\r\ntop: 0;; }",
+      fixed: "a { color: pink;;\r\ntop: 0;;}",
       description: "CRLF",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 9
     },
     {
       code: "a { color: pink;;;\ntop: 0;;; }",
+      fixed: "a { color: pink;;;\ntop: 0;;;}",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 10
     },
     {
       code: "a { color: pink;;;\r\ntop: 0;;; }",
+      fixed: "a { color: pink;;;\r\ntop: 0;;;}",
       description: "CRLF",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 10
     },
     {
       code: "a { color: pink;\ntop: 0;\n}",
+      fixed: "a { color: pink;\ntop: 0;}",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 8
     },
     {
       code: "a { color: pink;\ntop: 0;  }",
+      fixed: "a { color: pink;\ntop: 0;}",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 9
     },
     {
       code: "a { color: pink;\ntop: 0;\t}",
+      fixed: "a { color: pink;\ntop: 0;}",
       message: messages.rejectedBeforeMultiLine,
       line: 2,
       column: 8
     },
     {
       code: "a { color: pink;\ntop: 0;} b { color: red;\ntop: 0;\n}",
+      fixed: "a { color: pink;\ntop: 0;} b { color: red;\ntop: 0;}",
       message: messages.rejectedBeforeMultiLine,
       line: 3,
       column: 8
+    },
+    {
```

Comment:
Can you explain these tests, please?

I'd expected `a { color: pink;;;\ntop: 0; ;;}` and `a { color: pink;;;\ntop: 0;; ;}` to be accepted as neither has a newline before the closing brace.

`a { color: pink;;;\ntop: 0; ; ; }` does, but the spacing between the semicolons is transformed to `a { color: pink;;;\ntop: 0;;;}`. 

I understand these are edge-cases, and if these artifacts are due to an unavoidable implementation detail then I'm happy to stick with them... I'd just like to understand them :)

Reply:
`"never-multi-line"` option does not allow whitespace before closing brace.
It can be understood also from the test case of `"a { color: pink;\ntop: 0; }"`.

Also, when I tried, `"a { color: pink;;;\ntop: 0; ;;}"` and `"a { color: pink;;;\ntop: 0;; ;}"` is not allowed with this option.

Since error is not reported when `--fix` option is used, all errors need to be resolved,
`"a { color: pink;;;\ntop: 0; ; ; }"` is transformed to `"a { color: pink;;;\ntop: 0;;;}"`.